src/wallet/wallet2.h

```diff
@@ -1811,6 +1813,7 @@ namespace tools
     crypto::secret_key m_rpc_client_secret_key;
     rpc_payment_state_t m_rpc_payment_state;
     uint64_t m_credits_target;
+    bool m_enable_multisig;
```

Comment:
```suggestion
    bool m_multisig_enabled;
```
`enable` is a verb, `enabled` is an adjective